lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
Sorry, but I disagree. Most of the functions here were created with the aim for performance, and introducing `toBuffer` means certain expectations from the API users. I would appreciate if we could meet this expectations and make it as fast as we can.

Thanks!